docs/tutorial/in-app-purchases.md

```diff
@@ -0,0 +1,125 @@
+# In-App Purchase (macOS)
+
+## Preparing
+
+### Paid Applications Agreement
+If it's not already done, you’ll need to sign the Paid Applications Agreement and set up your banking and tax information in iTunes Connect. 
```

Comment:
"If it's not already done," might read more smoothly as "If you haven't already, "